README.md

```diff
@@ -40,7 +40,7 @@ This section deals with a full installation of WS Export on the local developmen
    For example, in Debian-based Linux distributions:
 
    ```console
-   apt install php-sqlite3 php-zip php-curl
+   apt install php-sqlite3 php-zip php-curl php-sysvsem
```

Comment:
Should also be added as a requirement in `composer.json`.

Reply:
This means that installation would break on Windows (no sysvsem module). Sadly it does not seems there is a way to set optional dependencies in composer.